### DISCUSSION TITLE
A Way to View the Coordinates of /homes

### DISCUSSION THREAD

**Discussion by Wh1rledPeas** [EssentialsX/Essentials#5561] [Ideas and Feature Suggestions]
I was playing on a server yesterday and someone asked if there was a way to see the coordinates for all of the homes they had set.

It would be nice to have a version of the /homes command that included the coordinates for each /sethome... 🤔

**Comment by Chew** [comment]
Sounds like something for the Adventure-era (when merged 😄), could easily add a way to hover over the `/homes` and see the coordinates, world, etc.

**Reply by NumbGnat** [reply]
Perhaps...

But I don't think it would be a big challenge to update the `/homes` command to output something like...

```
Spawn Village (1435, 64, 768)
Diamond Mine (24765, -54, 13218)
```
